robot/EDA/resources/locator.py

```diff
@@ -21,7 +21,7 @@
     "eda_settings": {
         "system_tab": "//a[contains(text(),'System')]",
         "affiliations_tab": "//a[contains(text(),'Affiliations')]",
-        "affiliations_check": "//span[text()='Specify Role for Created Affiliations:']/../following-sibling::div/descendant::img[@class = 'copy-start-date checked' and @alt='True']",
+        "affiliations_check": "//span[text()='Specify Role for Created Affiliations:']/../following-sibling::div/div/div/label/span/img[@class = 'copy-start-date checked' and @alt='True']",
```

Comment:
@dashattuck Just out of curiosity, do you know why Samsheeth's changes would have affected this? The affiliations settings are on a different tab than the settings he added...

Reply:
Earlier today he and I were debugging.  I modified this code to remove some robot errors.  I found that the modifications were good, so I checked them in.  The above diff looks correct.

Reply:
@robertbwatson I updated those files with @dashattuck.

Reply:
That's fine, I was just trying to understand how the changes in this PR caused problems. But I don't need to understand.